source/main.cpp

```diff
@@ -53,10 +54,31 @@ void PrintHelp();
 void PrintVersion();
 void GameLoop(PlayerInfo& player, Conversation& conversation, bool& debugMode);
 Conversation LoadConversation();
+int restOfMain(bool loadOnly, bool debugMode, Conversation conversation);
 #ifdef _WIN32
 void InitConsole();
 #endif
 
+struct restOfMainArgs
+{
+    bool debugMode;
+    bool loadOnly;
+    Conversation conversation;
+} args;
+int loadIndex = 0;
+
+void loadIter(void* data)
+{
+    bool done = GameData::LoadALittle(loadIndex);
+    if (!done) {
+        loadIndex++;
+        emscripten_async_call(loadIter, NULL, 20);
```

Comment:
this number doesn't need to be as big as 20ms, it could be 0 and the browser would still be responsive to scroll events etc.